src/test/java/org/folio/ebsconet/TestBase.java

```diff
@@ -11,16 +11,14 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.boot.test.web.server.LocalServerPort;
 import org.springframework.test.context.TestPropertySource;
-import org.springframework.transaction.annotation.EnableTransactionManagement;
-import org.springframework.util.SocketUtils;
+import org.springframework.test.util.TestSocketUtils;
 
```

Comment:
In spring boot, LocalServerPort changed to test package
![image](https://user-images.githubusercontent.com/113523904/213667200-cddbd908-5846-4df7-80de-fbadcaae2191.png)